cockatrice/src/carddatabase.cpp

```diff
@@ -340,6 +340,71 @@ void PictureLoader::processLoadQueue()
     }
 }
 
+int PictureLoader::getIdFromHtml(QString html, QString lang)
+{
+    QStringList langs = html.split("class=\"cardItem");
+    for(int i = 1; i < langs.size(); ++i)
+    {
+        const QString langHtml = langs.at(i);
+        if(langHtml.contains(lang,Qt::CaseInsensitive))
+        {
+            //qDebug() << langHtml;
+            int posStart = langHtml.indexOf("multiverseid=") + 13;
+            int posEnd = langHtml.indexOf("\">",posStart);
+            QString sId = langHtml.mid(posStart, posEnd-posStart);
+            return sId.toDouble();
+        }
+    }
+    return 0;
+}
+
+QMap<QString,QString> langDict;
+
+int PictureLoader::getTranslatedId(int currentId)
+{
+    if(langDict.isEmpty())
+    {
+        langDict["cs"] =  "繁體中文";
+        langDict["de"] =  "Deutsch";
+        langDict["es"] =   "Español";
+        langDict["fr"] =   "Français";
+        langDict["it"] =   "Italiano";
+        langDict["ja"] =   "日本語";
+        langDict["ko"] =   "한국어";
+        langDict["pt_BR"] =  "Português";
+        langDict["pt"] =     "Português";
+        langDict["ru"] =     "русский язык";
+        langDict["zh-Hans"] = "简体中文";
+    }
```

Comment:
the Map key is language settings related ( returned by settingsCache->getLang() ) and the values are gatherer related, used to search for muid translation (it the "Translated Language" column of the gathere language page)